### PR TITLE
Add support to external data field

### PR DESCRIPTION
**Pull request by mbencer**

### CODE REVIEW

**Comment by silee2** [comment]
Looks like this PR and #3257 have some overlapping

**Comment by postrational** [comment]
This PR needs to be merged first: https://github.com/NervanaSystems/ngraph/pull/3257

File: src/ngraph/frontend/onnx_import/onnx.cpp [hunk 1]
```diff
@@ -52,7 +52,32 @@ namespace ngraph
             } // namespace error
         }     // namespace detail
 
-        std::shared_ptr<Function> import_onnx_model(std::istream& sin, const Weights& weights)
+        static void update_external_data_paths(onnx::ModelProto& model_proto,
```
Comment (mbencer) [inline src/ngraph/frontend/onnx_import/onnx.cpp]: It is worth to consider if we want change onnx API to OPP version. In such case passing `model_path` as additional argument can be replaced by class field introduction.

**Comment by diyessi** [comment]
@mbencer please close if this is dead.

**Comment by mbencer** [comment]
> @mbencer please close if this is dead.

I suppose it will be continued

**Comment by diyessi** [comment]
Obsolete

**Pull request closed without merging**